build.rs

```diff
@@ -39,7 +39,7 @@ impl fmt::Debug for Error {
             Error::OutDir(env::VarError::NotPresent) => unreachable!(),
             Error::OutDir(env::VarError::NotUnicode(msg)) => msg.to_string_lossy().to_string(),
             Error::InvalidUserHooksDir(path) => {
-                format!("User hooks directory is not found or empty: {:?}", path)
+                format!("User hooks directory is not found or no executable file is found in the directory: {:?} \nDid you forget to make a hook script executable?", path)
```

Comment:
Would you avoid new line in error message? I'd like to suggest:

```rust
format!("User hooks directory is not found or no executable file is found in '{:?}'. Did you forget to make a hook script executable?", path)
```

Reply:
I was worried the path would be too long to look nice, but you're right and I've fixed it :)